drivers/led_strip/Kconfig.ws2812

```diff
@@ -8,36 +8,47 @@
 # https://wp.josh.com/2014/05/13/ws2812-neopixels-are-not-so-finicky-once-you-get-to-know-them/
 
 config WS2812_STRIP_SPI
-	bool "SPI driver"
+	bool "WS2812 LED strip SPI driver"
+	default y
+	depends on DT_HAS_WORLDSEMI_WS2812_SPI_ENABLED
 	select SPI if $(dt_compat_on_bus,$(DT_COMPAT_WORLDSEMI_WS2812_SPI),spi)
 	help
+	  Enable driver for WS2812 (and compatibles) LED strip that using SPI.
```

Comment:
I think you can remove "that" ^

Suggestion:
`Enable driver for WS2812 (and compatibles) LED strip controllers using SPI.`
Or:
`Enable SPI driver for WS2812 (and compatibles) LED strip controllers.`

This proposition also applies to the other help messages below.

Reply:
I applied your suggestion. Thank you.